undefined: prometheus.Handler

Dears,

I'm trying run 

`go build dovecot_export.go`

but I have an error

`./dovecot_exporter.go:219:28: undefined: prometheus.Handler`

Someone could help me ? How can I solve it ?

--- a/dovecot_exporter.go
+++ b/dovecot_exporter.go
@@ -25,6 +25,7 @@ import (
 	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
@@ -216,7 +217,7 @@ func main() {
 	exporter := NewDovecotExporter(*socketPath, strings.Split(*dovecotScopes, ","))
 	prometheus.MustRegister(exporter)
 
-	http.Handle(*metricsPath, prometheus.Handler())
+	http.Handle(*metricsPath, promhttp.Handler())
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		w.Write([]byte(`
 			<html>

